extension/js/background.js

```diff
@@ -27,11 +31,16 @@ chrome.omnibox.onInputChanged.addListener(omnibarHandler);
 chrome.omnibox.onInputEntered.addListener(acceptInput);
 chrome.runtime.onMessage.addListener(handleMessage);
 chrome.runtime.onInstalled.addListener(function (object) {
-    chrome.storage.local.get("shouldOpenTab", function(item) {
+    chrome.storage.local.get("shouldOpenTab", function (item) {
         if (Object.keys(item).length == 0) {
-            chrome.tabs.create({url: "assets/about.html"}, function (tab) {
-            });
-            chrome.storage.local.set({"shouldOpenTab": {"dontShow": true}})
+            chrome.tabs.create({
```

Comment:
is this (and many other changes) a choice or your editor autoformat the entire file?